tools/UpdateDotnetRuntime.ps1

```diff
@@ -130,24 +139,29 @@ function Update-CsprojFile([string] $path, $values) {
 }
 
 function Get-DotnetUpdate {
-    try {
-        $dotnetMetadataPath = "$PSScriptRoot/../DotnetRuntimeMetadata.json"
-        $nextChannel = (Get-Content $dotnetMetadataPath -Raw | ConvertFrom-Json).sdk.nextChannel
-        $latestSDKversion = [System.Management.Automation.SemanticVersion] (Invoke-RestMethod -Uri "http://aka.ms/dotnet/$nextChannel/Sdk/productVersion.txt" -ErrorAction Stop | ForEach-Object { $_.Trim() })
-        $currentVersion = [System.Management.Automation.SemanticVersion] (( Get-Content -Path "$PSScriptRoot/../global.json" -Raw | ConvertFrom-Json).sdk.version)
-
-        if ($latestSDKversion -gt $currentVersion) {
-            $shouldUpdate = $true
-            $newVersion = $latestSDKversion
-        } else {
+    if ($SDKVersionOverride) {
+        $shouldUpdate = $true
+        $newVersion = $SDKVersionOverride
```

Comment:
It might be better to just return early here, rather than push most of the function into an `else` clause

Reply:
Fixed